app/api/items/items.go

```diff
@@ -73,15 +76,42 @@ type permission struct {
 
 type itemChild struct {
 	// required: true
-	ItemID int64 `json:"item_id,string" sql:"column:child_item_id" validate:"set,child_item_id"`
+	ItemID int64 `json:"item_id,string" sql:"column:child_item_id" validate:"set"`
 	// default: 0
 	Order int32 `json:"order" sql:"column:child_order"`
+	// enum: Undefined,Discovery,Application,Validation,Challenge
+	// default: Undefined
+	// required: true
+	Category string `json:"category" validate:"oneof=Undefined Discovery Application Validation Challenge"`
+	// default: 1
+	ScoreWeight int8 `json:"score_weight"`
+	// Can be set to 'as_info' if `can_grant_view` != 'none' or to 'as_content' if `can_grant_view` >= 'content'.
+	// Defaults to 'as_info' (if `can_grant_view` != 'none') or 'none'.
+	// enum: none,as_info,as_content
+	ContentViewPropagation string `json:"content_view_propagation" validate:"oneof=none as_info as_content"`
+	// Can be set to 'as_is' (if `can_grant_view` >= 'solution') or 'as_content_with_descendants'
+	// (if `can_grant_view` >= 'content_with_descendants') or 'use_content_view_propagation'.
+	// Defaults to 'as_is' (if `can_grant_view` >= 'solution') or 'as_content_with_descendants'
+	// (if `can_grant_view` >= 'content_with_descendants') or 'use_content_view_propagation' (otherwise).
+	// enum: use_content_view_propagation,as_content_with_descendants,as_is
+	UpperViewLevelsPropagation string `json:"upper_view_levels_propagation" validate:"oneof=use_content_view_propagation as_content_with_descendants as_is"` // nolint:lll
+	// Can be set to true if `can_grant_view` >= 'solution_with_grant'.
+	// Defaults to true  if `can_grant_view` >= 'solution_with_grant'.
```

Comment:
I think ` Defaults to true  if ...` does not makes very clear that it is false otherwise.
So either you should add "false otherwise" or just say `Defaults to whether ...`